cirq-core/cirq/ops/measure_util.py

```diff
@@ -33,19 +33,28 @@ def measure_single_paulistring(
     pauli_observable: pauli_string.PauliString,
     key: Optional[Union[str, 'cirq.MeasurementKey']] = None,
 ) -> raw_types.Operation:
-    """Returns a single PauliMeasurementGate which measures the pauli observable
+    """Returns an operation, from `cirq.PauliMeasurementGate`, to measure the `pauli_observable`.
```

Comment:
nit: "from" is strange here.  Maybe "a `cirq.MeasurementGate`"?

Reply:
Yeah, I was originally planning to write:
``` 
"""Returns a `cirq.PauliMeasurementGate` to measure the `pauli_observable`.
```

but then there's the technicality that we are returning an operation and not a gate, which corresponds to applying the pauli measurement gate on a bunch of qubits. Do you have any recommendation here?

Reply:
Updated.